package.json

```diff
@@ -1,18 +1,26 @@
 {
   "name": "loopback-workspace",
   "version": "4.0.0-alpha.1",
-  "main": "index.js",
+  "main": "server/server.js",
+  "engines": {
+    "node": ">=4"
+  },
   "scripts": {
-    "test": "node .",
-    "lint": "eslint ."
+    "test": "eslint .",
+    "lint": "eslint .",
+    "posttest": "npm run lint"
   },
   "repository": {
     "type": "git",
     "url": "https://github.com/strongloop/loopback-workspace.git"
   },
   "license": "MIT",
   "dependencies": {
-    "loopback": "^3.0.0"
+    "chai": "^3.5.0",
```

Comment:
you should use dirty-chai too at some point, usually when you set up the expect helper. See https://github.com/strongloop/loopback-connector-kv-redis/blob/master/test/helpers/expect.js